db/migrate/20200514092553_move_katello_fields_from_hostgroups.katello.rb

```diff
@@ -0,0 +1,49 @@
+class MoveKatelloFieldsFromHostgroups < ActiveRecord::Migration[6.0]
```

Comment:
This migration is now failing in this way:

```
== 20200514092553 MoveKatelloFieldsFromHostgroups: migrating ================== 
rake aborted!                                                                                                                                                 
StandardError: An error has occurred, this and all later migrations canceled:   \
                                                                              
Validation failed: Content source with given ID not found                                                                                                     
```

I don't think unscoped is enough.. seems like it's related to User.current being nil. Your console testing is working fine because User.current == foreman_console_admin there